jax/_src/lax/lax.py

```diff
@@ -3663,7 +3663,7 @@ def _reduce_sum_shape_rule(operand, *, axes):
   return _reduce_op_shape_rule(operand, axes=axes)
 
 def _reduce_sum_transpose_rule(cotangent, operand, *, axes):
-  assert ad.is_undefined_primal(operand)
+  # assert ad.is_undefined_primal(operand)
```

Comment:
Did you want to comment out this assertion still?

If so, maybe add a comment explaining why?

Reply:
Oops, didn't mean to leave that in!

Reply:
Tyty fixed.